package.json

```diff
@@ -17,7 +18,9 @@
     "package": "cross-env NODE_ENV=production node -r babel-register -r babel-polyfill webpack/package.js",
     "package-all": "npm run package -- --all",
     "postinstall": "node node_modules/fbjs-scripts/node/check-dev-engines.js package.json",
+    "preinstall": "which electron || npm install electron@1.4.12",
```

Comment:
Well if you don't use Nix with daedalus, then it still needs to install `electron`. If you do use Nix, `preinstall` is a noop.